services/core-api/app/api/mines/response_models.py

```diff
@@ -105,6 +109,7 @@ def format(self, value):
         'issue_date': fields.DateTime(dt_format='iso8601'),
         'authorization_end_date': fields.DateTime(dt_format='iso8601'),
         'security_total': fields.Fixed(description='Currency', decimals=2),
+        'security_received_date': fields.DateTime(dt_format='iso8601'),
```

Comment:
we should have been able to get away from this iso8601 flag. Do we need this parameter?

Reply:
no, I added it for consistency with the other date fields. should it just be `fields.DateTime()`?

Reply:
I think that should work just as well yes.